src/components/app-detail.js

```diff
@@ -265,7 +265,4 @@ const mapDispatchToProps = dispatch => {
   }
 }
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(AppDetail)
+export default connect(mapStateToProps, mapDispatchToProps)(AppDetail)
```

Comment:
This commit was suggested by prettier code-formatter, moreover it is feasible as other files are following the same pattern.